manifest.json

```diff
@@ -510,7 +603,10 @@
               "dockerfile": "src/ubuntu/18.04/arm64v8",
               "os": "linux",
               "tags": {
-                "ubuntu-18.04-arm64v8-$(System:DockerfileGitCommitSha)-$(System:TimeStamp)": {}
+                "ubuntu-18.04-arm64v8-$(System:DockerfileGitCommitSha)-$(System:TimeStamp)": {},
+                "ubuntu-18.04-arm64v8": {
```

Comment:
Please revert these unnecessary manifest changes.  Since you aren't adding a dependency on these images the local tag is not needed.

Reply:
Reverted